documenu.py

```diff
@@ -0,0 +1,109 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+#This endpoint enables users to search for restaurants with the purpose of returning the selected restaurant's id for endpoint2: 
+# Users have filters like zipcode, address, etc..
+# parse through to get restaurant id
+def endpoint1():
+
+    #users search by the following parameters, which will be inputted from html form inputs:
+    restaurant_name = "Hard Rock Cafe"
+    state = "Georgia"
+    zip_code = "30308"
+    address = "International Boulevard Northeast"
+    select_number_of_results = "10"
+    
+    url = "https://documenu.p.rapidapi.com/restaurants/search/fields"
+    querystring = {"restaurant_name":restaurant_name, "address": address,"state":state,"zip_code":zip_code, "size":select_number_of_results}
+
+    headers = {
+        'x-api-key': os.getenv('x-api-key'),
+        'x-rapidapi-host': os.getenv("x-rapidapi-host-documenu"),
+        'x-rapidapi-key': os.getenv('x-rapidapi-key')
+        }
+    response = requests.request("GET", url, headers=headers, params=querystring)
+    Jresponse = response.json()
+    restaurant_id_parsed = Jresponse['data'][0]['restaurant_id']
+    return restaurant_id_parsed
+    
+
+restaurant_id = endpoint1()
+
+
+#this endpoint gets menu information about ONE restaurant 
+def endpoint2(restaurant_id):
```

Comment:
The name endpoint2() does not give much information about the function's purpose. Can you please provide a descriptive name that describes what the function does.